hikari/impl/entity_factory.py

```diff
@@ -2571,6 +2572,7 @@ def deserialize_command_interaction(
             resolved=resolved,
             target_id=target_id,
             app_permissions=permission_models.Permissions(app_perms) if app_perms else None,
+            entitlements=[self.deserialize_entitlement(entitlement) for entitlement in payload.get("entitlements", [])],
```

Comment:
I really dislike this line, and it should probably be brought out

```suggestion
            entitlements=[self.deserialize_entitlement(entitlement) for entitlement in payload.get("entitlements", ())],
```